docs/discover/search-for-relevance.asciidoc

```diff
@@ -0,0 +1,24 @@
+[[discover-search-for-relevance]]
+=== Search for relevance
```

Comment:
If you don't want this page to be nested under the "Search data" page, it needs to be bumped up a level:
```suggestion
== Search for relevance
```